### ISSUE TITLE
ModuleNotFoundError: No module named 'agents'

### ISSUE THREAD

**Issue by gonultasbu** [CPS-TUWien/racing_dreamer#11]
Hello,

While trying to run the training, I get the following error:
```
File "dream.py", line 16, in <module>
    from agents.gap_follower import GapFollower
ModuleNotFoundError: No module named 'agents'
```
I tried debugging the issue but have not been able to figure out where the GapFollower is imported. More specifically, where is the agents defined as a package or a python script?

**Comment by axelbr** [comment]
Yes, you are right, thanks for raising this issue. The reason for this is that the `agents` package was originally part of `racecar_gym`, where we eventually removed it. I recovered the class and will push it to the repository after I tested it. I need to fix the docker file as well. In the meantime, you can find it here: 
[https://github.com/axelbr/racecar_gym/blob/5ef58f71bb858a9c4952af5e8728d44c82d9de72/agents/gap_follower.py](https://github.com/axelbr/racecar_gym/blob/5ef58f71bb858a9c4952af5e8728d44c82d9de72/agents/gap_follower.py)

**Comment by gonultasbu** [comment]
Thanks for the answer Axel, there seems to be an unsolvable dependency issue with the `requirements.txt` as well. I believe it is also linked to this problem; where the tagged (therefore cloned+installed) version of `racecar_gym` is incompatible with this repository.

I have tried manually solving dependency conflicts one by one until I encountered this `agents` problem, I believe referencing to the right commit will solve that issue too.
